ZIGSIMPlus/Utils.swift

```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 import UIKit
+import SwiftOSC
```

Comment:
Is this necessary?

Reply:
ah nope, I just forgot to remove this line.
fixed in bc11b8190